tracim/command/database.py

```diff
@@ -113,10 +114,19 @@ def take_action(self, parsed_args: argparse.Namespace) -> None:
         setup_logging(config_uri)
         settings = get_appsettings(config_uri)
         engine = get_engine(settings)
+        app_config = CFG(settings)
+        app_config.configure_filedepot()
+
         if parsed_args.force:
-            print('Database deletion begin')
+            print('Database deletion begin.')
             DeclarativeBase.metadata.drop_all(engine)
-            print('Database deletion done')
+            print('Database deletion done.')
+            print('Cleaning depot begin.')
+            depot = DepotManager.get()
+            depot_files = depot.list()
+            for file in depot_files:
```

Comment:
do not use `file` as a variable name because it's a python keyword. Use `_file` or `file_` instead